py4DSTEM/process/diffraction/crystal_ACOM.py

```diff
@@ -76,65 +88,98 @@ def orientation_plan(
     # Handle the "auto" case first, since it works by overriding zone_axis_range,
     #   fiber_axis, and fiber_angles then using the regular parser:
     if isinstance(zone_axis_range, str) and zone_axis_range == "auto":
-        from pymatgen.symmetry.analyzer import SpacegroupAnalyzer
-        from pymatgen.core.structure import Structure
+        # from pymatgen.symmetry.analyzer import SpacegroupAnalyzer
+        # from pymatgen.core.structure import Structure
 
+        # initialize structure
         structure = Structure(
             self.lat_real, self.numbers, self.positions, coords_are_cartesian=False
         )
 
-        pointgroup = SpacegroupAnalyzer(structure).get_point_group_symbol()
-        self.pointgroup = pointgroup
+        # pointgroup = SpacegroupAnalyzer(structure).get_point_group_symbol()
+        # self.pointgroup = pointgroup
+        self.pointgroup = SpacegroupAnalyzer(structure)
```

Comment:
Should this be in a try statement, as it is liked to pymatgen? If it fails it could raise a warning that pymatgen is required.

Reply:
or it could be an if statement if the `pymatgen_available` flag is set before this.